core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -401,6 +402,9 @@ export class LoggedInUser extends BaseUser {
     await this.page.$eval(weCannotContentId, element =>
       element.getElementsByTagName('a')[0].click()
     );
+    await this.page.waitForNavigation({waitUntil: ['load', 'networkidle2']});
+    await this.page.waitForSelector(_420millionPageHeaderSelector);
```

Comment:
Do you actually need this line, or is the previous one alone sufficient?

Reply:
Previous one alone is good enough to handle this.